.github/workflows/build.yml

```diff
@@ -0,0 +1,66 @@
+name: Docker Build and Push
+
+on:
+  release:
+    types: [released, prereleased]
+  push:
+    branches:
+      - master
+
+      # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get the version
+        if: ${{ github.event_name == 'release' }}
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push release
+        if: ${{ github.event_name == 'release' && github.event.action == 'released' }}
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: |
+            jaydwee/koalabot:latest
+            jaydwee/koalabot:${{ steps.get_version.outputs.VERSION }}
```

Comment:
do you think there's value in creating a koalabot account on dockerhub and tag the image there instead of using your own personal account?

Reply:
I might look into it in the future, but for now it should be fine, especially if we are the only ones using the docker image